homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -3,11 +3,16 @@
 
 from homeassistant.components.device_tracker import DeviceScanner
 
-from . import DATA_ASUSWRT
+from .const import DATA_ASUSWRT
 
 _LOGGER = logging.getLogger(__name__)
 
 
+async def async_setup_entry(hass, entry, async_add_entities):
```

Comment:
You should migrate the device tracker to the config entry friendly format, UniFi integration is one example

Reply:
I already did this, but was asked to me to do this with multiple PR, for this reason I'm merging to dedicated branch and not in dev.